src/components/Navbar/styles.scss

```diff
@@ -1,4 +1,4 @@
-@import '../../styles/main.scss';
+@import "../../styles/main.scss";
```

Comment:
Normally we use a single quote after SCSS `@import` notation.
Ref: https://sass-lang.com/documentation/at-rules/import